docfiles/head.html

```diff
@@ -1,8 +1,8 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
 <meta name="viewport" content="width=device-width,height=device-height,user-scalable=no,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0">
 
-<link rel="stylesheet" href="/doccdn/semantic.css" />
-<link rel="stylesheet" href="/doccdn/blockly.css" />
+<link rel="stylesheet" href="/blb/semantic.css" />
+<link rel="stylesheet" href="/blb/blockly.css" />
```

Comment:
How does this work for index.html? 

Do we just need to add it to this list in cli.ts: 
```

    let replFiles = [
        "index.html",
        "embed.js",
        "run.html",
        "docs.html",
        "siminstructions.html",
        "release.manifest",
        "worker.js",
        "tdworker.js",
        "monacoworker.js",
        "simulator.html",
        "sim.manifest",
        "sim.webmanifest",
    ]

```